layers/meta-balena-beaglebone/conf/layer.conf

```diff
@@ -26,9 +26,23 @@ PREFERRED_PROVIDER_u-boot:beagleboard-xm = "u-boot"
 KERNEL_IMAGETYPES:beaglebone-ai64 = "Image"
 KERNEL_CLASSES:beaglebone-ai64 = "kernel-uimage"
 
+PREFERRED_PROVIDER_virtual/kernel:beagleplay = "linux-bb.org"
```

Comment:
Sure. Will document this. Although it now looks like we *may* be able to simply use the `linux-beagleboard` provider.

Reply:
@floion - looking at the structure of the `linux-beagleboard_5.10.bb` recipe, we will have to make some additional changes to it (or the *.bbappend file) in order to **not** use the `linux-bb.org` recipes. 

Is it ok if we continue to use the `linux-bb.org` recipes for the beagleplay ? I will document the specific reasons (as requested).

Reply:
@delhiryder if there are reasons to switch to `linux-bb.org` instead of `linux-beagleboard` I am fine with the change as long as we document why we need to use the new one and why we can't stick to using the old one.